sqlglot/expressions.py

```diff
@@ -5506,6 +5506,11 @@ class ParseJSON(Func):
     is_var_len_args = True
 
 
+class TryParseJSON(Func):
+    # https://docs.snowflake.com/en/sql-reference/functions/try_parse_json
+    _sql_names = ["TRY_PARSE_JSON"]
+
```

Comment:
Iirc in Snowflake we encode the `TRY` variant of a func through a `safe` arg in the expression; For instance, in this case I see that there's `PARSE_JSON` and `TRY_PARSE_JSON`, maybe we could implement `exp.ParseJson(this, safe)`?

Reply:
Sure